cmd/access/node_builder/access_node_builder.go

```diff
@@ -1839,6 +1848,41 @@ func (builder *FlowAccessNodeBuilder) Build() (cmd.Node, error) {
 			return builder.RequestEng, nil
 		})
 
+	if builder.versionControlEnabled {
+		builder.Component("version control", func(node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+			ver, err := build.Semver()
+			if err != nil {
+				err = fmt.Errorf("could not load node version for version control. "+
+					"version %s is not semver compliant: %w. Make sure a valid semantic version is provided in the VERSION environment variable", build.Version(), err)
+
+				return nil, err
+			}
+
+			latestFinalizedBlock, err := node.State.Final().Head()
+			if err != nil {
+				return nil, fmt.Errorf("could not get latest finalized block: %w", err)
+			}
+
+			versionControl, err := version.NewVersionControl(
+				builder.Logger,
+				node.Storage.VersionBeacons,
+				ver,
+				builder.FinalizedRootBlock.Header.Height,
+				latestFinalizedBlock.Height,
```

Comment:
looks like there's an existing variable that has this
```suggestion
				builder.LastFinalizedHeader.Height,
```